service/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactory.java

```diff
@@ -274,11 +279,28 @@ public Deployable definition(DefinitionContext definitionContext) {
               .withSystemAssignedManagedServiceIdentity()
               .defineAgentPool(
                   nameGenerator.nextName(ResourceNameGenerator.MAX_AKS_AGENT_POOL_NAME_LENGTH))
-              .withVirtualMachineSize(ContainerServiceVMSizeTypes.STANDARD_A2_V2)
-              .withAgentPoolVirtualMachineCount(1)
-              .withAgentPoolMode(
-                  AgentPoolMode.SYSTEM) // TODO VM Size? Pool Machine count? AgentPoolMode?
-              .withVirtualNetwork(vNetwork.id(), Subnet.AKS_SUBNET.name())
+              .withVirtualMachineSize(
+                  ContainerServiceVMSizeTypes.fromString(
+                      parametersResolver.getValue(ParametersNames.AKS_MACHINE_TYPE.name())))
```

Comment:
If we are adding new parameters, it would be helpful to update the README section that documents the parameters for each lz definition.